locations/spiders/ameriprise.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from scrapy.spiders import SitemapSpider
+
+from locations.google_url import extract_google_position
+from locations.items import Feature
+from locations.linked_data_parser import LinkedDataParser
+
+
+class AmeripriseSpider(SitemapSpider):
+    name = "ameriprise"
```

Comment:
Should this be `ameriprise_us`? i.e. will there be entries from other countries?

Reply:
Can we change this to `AmeripriseUSSpider` and `ameriprise_us` name? I think we're trying to move towards this on spiders that only produce data for a single country.